tests/memgraph/test_query_builder.py

```diff
@@ -1340,3 +1340,47 @@ class Follows_test(Relationship, type="FOLLOWS"):
         query_builder.execute()
 
     mock.assert_called_with(expected_query)
+
+
+def test_wShortest_bound():
+    weighted_shortest = WeightedShortestPath(upper_bound=10, weight_property="weight")
+
+    query_builder = (
+        QueryBuilder()
+        .match()
+        .node(variable="a", id=723)
+        .to(variable="edge_list", directed=False, algorithm=weighted_shortest)
+        .node(variable="b", id=882)
+        .return_()
+    )
+
+    expected_query = (
+        " MATCH (a {id: 723})-[edge_list * wShortest 10 (e, v | e.weight) total_weight]-(b {id: 882}) RETURN * "
+    )
+
+    with patch.object(Memgraph, "execute_and_fetch", return_value=None) as mock:
+        query_builder.execute()
+
+    mock.assert_called_with(expected_query)
+
+
+def test_wShortest_filter_label():
```

Comment:
Add test for when `weight_property` is `e.weight()` and when there is no `upper_bound`.